sdk/src/rent.rs

```diff
@@ -67,23 +73,44 @@ mod tests {
 
     #[test]
     fn test_due() {
-        let rent = Rent::default();
+        let mut zero_rent = Rent::default();
+        zero_rent.lamports_per_byte_year = 0;
 
         assert_eq!(
-            rent.due(0, 1, 1.0),
+            zero_rent.due(0, 1, 1.0),
```

Comment:
@ParthDesai: the first test will fail if you change DEFAULT_LAMPORTS_PER_BYTE_YEAR, now....

you should be able to modify the existing test to succeed no matter what the value of DEFAULT_LAMPORTS_PER_BYTE_YEAR is set to...

Reply:
done